linkerd/app/core/src/metric_labels.rs

```diff
@@ -50,11 +50,11 @@ struct Authority<'a>(&'a http::uri::Authority);
 
 // === impl CtlLabels ===
 
-impl From<control::ControlAddr> for ControlLabels {
-    fn from(c: control::ControlAddr) -> Self {
+impl<'c> From<&'c control::ControlAddr> for ControlLabels {
+    fn from(c: &'c control::ControlAddr) -> Self {
```

Comment:
nit/TIOLI: we could also use `'_` here if you like:
```suggestion
impl From<&'_ control::ControlAddr> for ControlLabels {
    fn from(c: &control::ControlAddr) -> Self {
```